backend/src/main/java/ca/bc/gov/backendstartapi/entity/idclass/ActiveOrchardSeedPlanningUnitId.java

```diff
@@ -0,0 +1,19 @@
+package ca.bc.gov.backendstartapi.entity.idclass;
+
+import ca.bc.gov.backendstartapi.entity.ActiveOrchardSeedPlanningUnit;
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+/** Composite key for {@link ActiveOrchardSeedPlanningUnit}. */
+@Data
+@NoArgsConstructor(access = AccessLevel.PACKAGE, force = true)
```

Comment:
The `AccessLevel` could be `PRIVATE`. And why use `force = true`?

Reply:
I let myself go with the annotations. `force` won't be needed here.